subworkflows/local/compute_gene_scores_heatmap/main.nf

```diff
@@ -0,0 +1,48 @@
+//
+// Compute gene scores and plot heatmap
+//
+
+//
+// MODULES
+//
+include { COMPUTE_GENE_SCORES     } from '../../../modules/local/compute_gene_scores'
+include { CREATE_GENE_HEATMAP     } from '../../../modules/local/create_gene_heatmap'
+
+workflow COMPUTE_GENE_SCORES_HEATMAP {
+    take:
+    normalized_counts          // channel: [ meta, normalized_counts.tsv ]
+    annotated_endo_data        // channel: [ meta, annotated_endo_data.tsv ]
+    ch_gene_score_yaml         // channel: [ file(gene_score_yaml.yaml) ]
+    ch_heatmap_genes_to_filter // channel: [ file(heatmap_genes_to_filter.yaml) ]
+
+    main:
+    ch_versions = Channel.empty()
+    ch_multiqc_files = Channel.empty()
+
+    //
+    // MODULE: Compute gene scores for supplied YAML gene score file
+    //
+    COMPUTE_GENE_SCORES(
+        normalized_counts,
+        ch_gene_score_yaml
+    )
+    ch_versions      = ch_versions.mix(COMPUTE_GENE_SCORES.out.versions)
+    ch_multiqc_files = ch_multiqc_files.mix(COMPUTE_GENE_SCORES.out.scores_for_mqc.map{ meta, file-> file }.collect())
+
+    //
+    // MODULE: Compute gene-count heatmap for MultiQC report based on annotated (ENDO) counts
+    //
+    if(!params.skip_heatmap){
```

Comment:
I think we can't have `params` inside subworkflows now. They should be only in the main workflow. There was a guideline somewhere written by Ben but I can't find it now.

Reply:
Makes sense, if this was an nf-core subworkflow this would be an issue. I don't think we will be migrating this subworkflow anytime soon, but since I already formatted it all, Ill do this too.